### PR TITLE
Adds manual shuttle landing

### PR DESCRIPTION
**Pull request by Anton-Kr**
You can now manually land shuttles. For now can only do this on planets, but plan to expand it to everything once this passes and gets settled in. Code taken from Nebula. Shouldn't break anything.

### CODE REVIEW

**Comment by Alanii** [comment]
You should make a mp4 of this. I don't really know what this means.

**Comment by Anton-Kr** [comment]
@Alanii done

https://user-images.githubusercontent.com/6289191/103475784-5576d000-4d7e-11eb-8506-00d6804e9bf3.mp4

**Comment by Anton-Kr** [comment]
Not that it will be, but please DNM until I fix this requiring master level piloting

File: code/modules/overmap/ships/computers/shuttle.dm [hunk 1]
```diff
@@ -39,8 +57,109 @@
...
 		else
-			to_chat(usr,"<span class='warning'>No valid landing sites in range.</span>")
+			to_chat(usr, SPAN_WARNING("No valid landing sites in range."))
 		possible_d = shuttle.get_possible_destinations()
 		if(CanInteract(usr, GLOB.default_state) && (D in possible_d))
 			shuttle.set_destination(possible_d[D])
 		return TOPIC_REFRESH
+
+	if(href_list["manual_landing"])
+		if(user.skill_check(SKILL_PILOT, SKILL_PROF))
```
Comment (BurpleBineapple) [inline code/modules/overmap/ships/computers/shuttle.dm]: This is what you need to change to allow manual landing for other skills. I would have it at SKILL_ADEPT so it's experienced or master only.

**Review by BurpleBineapple — CHANGES_REQUESTED** [review]

**Comment by Kaksisilma** [comment]
Aroma

**Comment by BurpleBineapple** [comment]
After further consideration, this might be something we should take upstream first. It touches a lot of core files and may cause conflicts when updating in the future.

If you don't want to redo this PR for baystation, I can do it for you and credit you for the original port. If baystation decides they don't want to add this, we can always go ahead and merge this then.

**Comment by Anton-Kr** [comment]
Skill changed to experienced and above.  

@BurpleBineapple I'm gonna PR this on Bay and see what comes of it. I'd honestly expect their review to take a while, and if we plan on implementing this regardless of if Bay does it, this could probably be merged when the Bay PR is up, unless there are some other reasons not to that are byond me.

**Review by BurpleBineapple — APPROVED** [review]
Since bay didn't seem interested in the idea, approving this here

**Pull request merged**